### PR TITLE
[event]mysterious chagllengers, field trip, lost at sea

### PR DESCRIPTION
**Pull request by sodaMelon**
# 미스터리한 도전자
image

# 현장학습
image

# 바다위의 미아
image

### CODE REVIEW

File: src/locales/ko/mystery-encounters/field-trip-dialogue.json [hunk 1]
```diff
@@ -1,31 +1,31 @@
 {
-  "intro": "It's a teacher and some school children!",
-  "speaker": "Teacher",
-  "intro_dialogue": "Hello, there! Would you be able to\nspare a minute for my students?$I'm teaching them about Pokémon moves\nand would love to show them a demonstration.$Would you mind showing us one of\nthe moves your Pokémon can use?",
-  "title": "Field Trip",
-  "description": "A teacher is requesting a move demonstration from a Pokémon. Depending on the move you choose, she might have something useful for you in exchange.",
-  "query": "Which move category will you show off?",
+  "intro": "선생님과 어린 학생들이다!",
+  "speaker": "선생님",
+  "intro_dialogue": "안녕하세요! 저희 학생들에게 잠시\n시간 좀 내주실 수 있을까요?$포켓몬의 기술에 대해서 가르치고 있는데\n실제 기술을 보게되면 좋아할 것 같아서요.$괜찮으시면 저희에게 포켓몬이\n사용할 수 있는 기술을 보여주실래요?",
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/field-trip-dialogue.json]:
```suggestion
  "intro_dialogue": "안녕하세요! 우리 학생들에게 잠시\n시간 좀 내주실 수 있을까요?$포켓몬 기술에 대해서 가르치고 있는데\n실제로 기술을 보면 좋아할 것 같아서요.$괜찮으시면 포켓몬이\n사용할 수 있는 기술을 보여주실래요?",
```

File: src/locales/ko/mystery-encounters/field-trip-dialogue.json [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+  "title": "현장 학습",
+  "description": "학교 선생님이 포켓몬의 기술 시연을 부탁했다. 선택한 기술에 따라, 선생님이 당신에게 유용한 것을 대가로 줄 수도 있을 것 같다.",
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/field-trip-dialogue.json]:
```suggestion
  "description": "학교 선생님이 포켓몬의 기술 시연을 부탁했다. 선택한 기술에 따라, 선생님이 뭔가 쓸 만한 걸 줄 수도 있을 것 같다.",
```

File: src/locales/ko/mystery-encounters/field-trip-dialogue.json [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
-  "second_option_prompt": "Choose a move for your Pokémon to use.",
-  "incorrect": "...$That isn't a {{moveCategory}} move!\nI'm sorry, but I can't give you anything.$Come along children, we'll\nfind a better demonstration elsewhere.",
-  "incorrect_exp": "Looks like you learned a valuable lesson?$Your Pokémon also gained some experience.",
-  "correct": "Thank you so much for your kindness!\nI hope these items might be of use to you!",
-  "correct_exp": "{{pokeName}} also gained some valuable experience!",
-  "status": "Status",
-  "physical": "Physical",
-  "special": "Special"
+  "second_option_prompt": "포켓몬이 사용할 기술을 골라주세요.",
+  "incorrect": "...$이건 {{moveCategory}} 기술이 아니네요!\n죄송하지만, 아무것도 드릴 수 없어요.$여러분 이리오세요. 다른 곳에서\n더 좋은 기술 시연을 찾아서 보여드릴게요.",
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/field-trip-dialogue.json]:
```suggestion
  "incorrect": "...$이건 {{moveCategory}} 기술이 아닌데요!\n죄송하지만, 저희는 이만 가볼게요.$얘들아 이리오렴. 다른 곳에서는\n제대로된 기술 시연을 볼 수 있을거야.",
```

File: src/locales/ko/mystery-encounters/field-trip-dialogue.json [hunk 4]
```diff
@@ -1,31 +1,31 @@
...
+  "incorrect_exp": "당신도 가치있는 실습(?)에서 배운 것 같다.$포켓몬도 약간의 경험치를 얻었다.",
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/field-trip-dialogue.json]:
```suggestion
  "incorrect_exp": "좋은 경험(?)을 한 것 같다.$포켓몬도 약간의 경험치를 얻었다.",
```

File: src/locales/ko/mystery-encounters/field-trip-dialogue.json [hunk 5]
```diff
@@ -1,31 +1,31 @@
...
+  "correct": "친절하게 보여주셔서 감사합니다!\n이 아이템이 도움이 되시길 바랄게요!",
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/field-trip-dialogue.json]:
```suggestion
  "correct": "멋진 시연 감사합니다!\n이 아이템이 도움이 되시길 바랄게요!",
```

File: src/locales/ko/mystery-encounters/field-trip-dialogue.json [hunk 6]
```diff
@@ -1,31 +1,31 @@
...
+  "correct_exp": "{{pokeName}}도 약간의 가치있는 경험치를 얻었다!",
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/field-trip-dialogue.json]:
```suggestion
  "correct_exp": "{{pokeName}}도 좋은 경험을 쌓았다!",
```

**Review by KimJeongSun — CHANGES_REQUESTED** [review]
valuable 을 이질감이 적은 `좋은` 으로 번역하면서 뉘앙스를 살려봤습니다.

File: src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json [hunk 7]
```diff
@@ -1,28 +1,28 @@
 {
-  "intro": "Wandering aimlessly through the sea, you've effectively gotten nowhere.",
-  "title": "Lost at Sea",
-  "description": "The sea is turbulent in this area, and you're running out of energy.\nThis is bad. Is there a way out of the situation?",
-  "query": "What will you do?",
+  "intro": "목표없이 바다를 헤메다보니, 정말로 당신은 아무 데도 갈 수 없게 되었다.",
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json]:
```suggestion
  "intro": "하염없이 바다를 떠돌다, 더이상 헤맬 곳조차 남지 않게 되었다.",
```

File: src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json [hunk 8]
```diff
@@ -1,28 +1,28 @@
...
+  "title": "바다 위의 미아",
+  "description": "이 지역의 바다는 난기류다. 그리고 당신은 에너지가 부족한 상태다.\n난감한 상황이다. 이 상황을 벗어날 수 있는 방법이 있을까?",
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json]:
```suggestion
  "description": "이곳은 미지의 바다다. 그리고 점점 기력이 떨어지고 있다.\n난감한 상황이다. 이 상황을 벗어날 방법이 있을까?",
```

File: src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json [hunk 9]
```diff
@@ -1,28 +1,28 @@
...
-      "label": "{{option1PrimaryName}} Might Help",
-      "label_disabled": "Can't {{option1RequiredMove}}",
-      "tooltip": "(+) {{option1PrimaryName}} saves you\n(+) {{option1PrimaryName}} gains some EXP",
-      "tooltip_disabled": "You have no Pokémon to {{option1RequiredMove}} on",
-      "selected": "{{option1PrimaryName}} swims ahead, guiding you back on track.${{option1PrimaryName}} seems to also have gotten stronger in this time of need!"
+      "label": "{{option1PrimaryName}}의 도움",
+      "label_disabled": "{{option1RequiredMove}} 사용 불가",
+      "tooltip": "(+) {{option1PrimaryName}}[[가]] 당신을 구함\n(+) {{option1PrimaryName}} 경험치 획득",
+      "tooltip_disabled": "{{option1RequiredMove}}[[를]] 사용할 수 있는 포켓몬이 없다.",
+      "selected": "{{option1PrimaryName}}[[가]] 앞장서 헤엄치며, 돌아가는 길을 안내했다.${{option1PrimaryName}}[[는]] 역시 이번 기회로 더 강해진 것 같다!"
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json]:
```suggestion
      "selected": "{{option1PrimaryName}}[[가]] 앞장서 헤엄치며, 돌아가는 길을 안내했다.${{option1PrimaryName}} 또한 이번 기회로 더 강해진 것 같다!"
```

File: src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json [hunk 10]
```diff
@@ -1,28 +1,28 @@
...
-      "label": "{{option2PrimaryName}} Might Help",
-      "label_disabled": "Can't {{option2RequiredMove}}",
-      "tooltip": "(+) {{option2PrimaryName}} saves you\n(+) {{option2PrimaryName}} gains some EXP",
-      "tooltip_disabled": "You have no Pokémon to {{option2RequiredMove}} with",
-      "selected": "{{option2PrimaryName}} flies ahead of your boat, guiding you back on track.${{option2PrimaryName}} seems to also have gotten stronger in this time of need!"
+      "label": "{{option2PrimaryName}}의 도움",
+      "label_disabled": "{{option2RequiredMove}} 사용 불가",
+      "tooltip": "(+) {{option2PrimaryName}}[[가]] 당신을 구함\n(+) {{option2PrimaryName}} 경험치 획득",
+      "tooltip_disabled": "{{option2RequiredMove}}[[를]] 사용할 수 있는 포켓몬이 없다.",
+      "selected": "{{option2PrimaryName}}[[가]] 보트 앞에서 날아가며, 돌아가는 길을 안내했다.${{option2PrimaryName}}[[는]] 이번 기회로 더 강해진 것 같다!"
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json]:
```suggestion
      "selected": "{{option2PrimaryName}}[[가]] 보트 앞에서 날아가며, 돌아가는 길을 안내했다.${{option2PrimaryName}} 또한 기회로 더 강해진 것 같다!"
```

File: src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json [hunk 11]
```diff
@@ -1,28 +1,28 @@
...
     },
     "3": {
-      "label": "Wander Aimlessly",
-      "tooltip": "(-) Each of your Pokémon lose {{damagePercentage}}% of their total HP",
-      "selected": "You float about in the boat, steering without direction until you finally spot a landmark you remember.$You and your Pokémon are fatigued from the whole ordeal."
+      "label": "목표없이 방황한다",
+      "tooltip": "(-) 모든 포켓몬의 HP가 {{damagePercentage}}%만큼 감소했다.",
+      "selected": "마침내 기억하는 랜드매크를 발견할 때까지, 당신은 정처없이 배를 조종하며 떠돌아다닌다.$포켓몬과 당신은 거대한 시련에 지쳐버렸다."
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json]:
```suggestion
      "selected": "마침내 기억하는 곳에 다다를 때까지, 정처없이 배를 타고 떠돌아다닌다.$길고 긴 표류에 포켓몬과 당신 모두 지쳐버렸다."
```

File: src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json [hunk 12]
```diff
@@ -1,28 +1,28 @@
...
     }
   },
-  "outro": "You are back on track."
+  "outro": "당신은 다시 길 위로 돌아왔다."
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/lost-at-sea-dialogue.json]:
```suggestion
  "outro": "다시 여정으로 돌아왔다."
```

File: src/locales/ko/mystery-encounters/mysterious-challengers-dialogue.json [hunk 13]
```diff
@@ -1,22 +1,22 @@
 {
-  "intro": "Mysterious challengers have appeared!",
-  "title": "Mysterious Challengers",
-  "description": "If you defeat a challenger, you might impress them enough to receive a boon. But some look tough, are you up to the challenge?",
-  "query": "Who will you battle?",
+  "intro": "미스터리한 도전자들이 나타났다!",
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/mysterious-challengers-dialogue.json]:
```suggestion
  "intro": "수수께끼의 도전자들이 나타났다!",
```

File: src/locales/ko/mystery-encounters/mysterious-challengers-dialogue.json [hunk 14]
```diff
@@ -1,22 +1,22 @@
...
+  "title": "미스터리한 도전자",
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/mysterious-challengers-dialogue.json]:
```suggestion
  "title": "수수께끼의 도전자",
```

File: src/locales/ko/mystery-encounters/mysterious-challengers-dialogue.json [hunk 15]
```diff
@@ -1,22 +1,22 @@
...
-      "label": "The Mightiest Foe",
-      "tooltip": "(-) Brutal Battle\n(+) Great Rewards"
+      "label": "가장 강한 도전자",
+      "tooltip": "(-) 악랄한 배틀\n(+) 훌륭한 보상"
     },
-    "selected": "The trainer steps forward..."
+    "selected": "트레이너가 앞으로 나아간다..."
   },
-  "outro": "The mysterious challenger was defeated!"
+  "outro": "미스터리한 도전자에게 승리했다!"
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/mysterious-challengers-dialogue.json]:
```suggestion
  "outro": "수수께끼의 도전자를 물리쳤다!"
```

File: src/locales/ko/mystery-encounters/mysterious-challengers-dialogue.json [hunk 16]
```diff
@@ -1,22 +1,22 @@
...
+      "tooltip": "(-) 어려운 배틀\n(+) 좋은 보상"
     },
     "3": {
-      "label": "The Mightiest Foe",
-      "tooltip": "(-) Brutal Battle\n(+) Great Rewards"
+      "label": "가장 강한 도전자",
+      "tooltip": "(-) 악랄한 배틀\n(+) 훌륭한 보상"
     },
-    "selected": "The trainer steps forward..."
+    "selected": "트레이너가 앞으로 나아간다..."
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/mysterious-challengers-dialogue.json]:
```suggestion
    "selected": "선택한 트레이너가 앞으로 걸어나온다..."
```

File: src/locales/ko/mystery-encounters/mysterious-challengers-dialogue.json [hunk 17]
```diff
@@ -1,22 +1,22 @@
 {
-  "intro": "Mysterious challengers have appeared!",
-  "title": "Mysterious Challengers",
-  "description": "If you defeat a challenger, you might impress them enough to receive a boon. But some look tough, are you up to the challenge?",
-  "query": "Who will you battle?",
+  "intro": "미스터리한 도전자들이 나타났다!",
+  "title": "미스터리한 도전자",
+  "description": "도전자에게 승리한다면, 보상 받을만큼 깊은 인상을 남길 수 있다. 몇몇 도전자들은 꽤 강해보이기도 한다. 도전을 받아줄까?",
```
Comment (KimJeongSun) [inline src/locales/ko/mystery-encounters/mysterious-challengers-dialogue.json]:
```suggestion
  "description": "도전자에게 승리해서 깊은 인상을 남긴다면, 보상을 받을 수 있을 것 같다. 몇몇 도전자들은 꽤 강해보인다. 도전을 받아줄까?",
```

**Review by KimJeongSun — CHANGES_REQUESTED** [review]

File: src/locales/ko/mystery-encounters/field-trip-dialogue.json (hunk 3 above)
Comment (sodaMelon) [reply src/locales/ko/mystery-encounters/field-trip-dialogue.json]:
```suggestion
  "incorrect": "...$이건 {{moveCategory}} 기술이 아닌데요!\n죄송하지만, 저희는 이만 가볼게요.$얘들아 이리오렴. 다른 곳에서는\n제대로된 기술 시연을 볼 수 있을거야.",
```

**Comment by KimJeongSun** [comment]
https://github.com/user-attachments/assets/d32a27b6-0be5-4c86-a272-a7f6ae5a9ec2


https://github.com/user-attachments/assets/2fe95eff-d1c2-4eb5-9fea-41d7391167b0


https://github.com/user-attachments/assets/65c68940-12bd-4ae4-8faf-5369a93174d7

**Pull request merged**